chore: Supports `options` in read and list for `stream_processor` (#2526)
mongodb/terraform-provider-mongodbatlas: docs/data-sources/stream_processor.md

### Read-Only

- `id` (String) Unique 24-hexadecimal character string that identifies the stream processor.
- `options` (Attributes) Optional configuration for the stream processor. (see [below for nested schema](#nestedatt--options))
- `pipeline` (String) Stream aggregation pipeline you want to apply to your streaming data.
- `state` (String) The state of the stream processor.
- `stats` (String) The stats associated with the stream processor.

<a id="nestedatt--options"></a>
### Nested Schema for `options`

Read-Only:

- `dlq` (Attributes) Dead letter queue for the stream processor. (see [below for nested schema](#nestedatt--options--dlq))

<a id="nestedatt--options--dlq"></a>
### Nested Schema for `options.dlq`

Read-Only:

> - `coll` (String) Name of the collection that will be used for the DLQ.

Reviewer: ```suggestion
- `coll` (String) Name of the collection to use for the DLQ.
```

> - `connection_name` (String) Connection name that will be used to write DLQ messages to. Has to be an Atlas connection.

Reviewer: ```suggestion
- `connection_name` (String) Name of the connection to write DLQ messages to. Must be an Atlas connection.
```

> - `db` (String) Name of the database that will be used for the DLQ.

Reviewer: ```suggestion
- `db` (String) Name of the database to use for the DLQ.
```
